docs/background/disaster-recovery.md

```diff
@@ -0,0 +1,54 @@
+# Disaster recovery
+
+When you go on to create a new server in {{brand}}, you will probably
+notice an option named **Disaster recovery**. That particular option is
+enabled by default.
+
+![Disaster recovery is enabled by default for new
+servers](assets/disaster-recovery-option-on-by-default.png)
+
+Even if you choose to disable it for a particular server, keep in mind
+that you have the option to enable it at a later time.
+
+![The disaster recovery feature can be manually activated for any
+server](assets/disaster-recovery-option-manual-activation.png)
+
+In the following, we explain what this option does, how it works in the
+background, and why you should consider enabling it.
+
+## What it is
+
+The *Disaster Recovery* feature, DR for short, is managed from the
+{{gui}} only and applies to servers and volumes that use our
+[Ceph](https://ceph.io) backend.
```

Comment:
The only way for this to _not_ be the case would be if a server uses an `s` flavor (see https://docs.cleura.cloud/reference/flavors/#compute-tiers), so let's explain that here. Any server that uses another flavor, and any volume, does use Ceph.

Reply:
Got it!